lib/code_corps/stripe/adapters/stripe_customer.ex

```diff
@@ -1,11 +1,29 @@
 defmodule CodeCorps.Stripe.Adapters.StripeCustomer do
   import CodeCorps.MapUtils, only: [rename: 3, keys_to_string: 1]
 
-  def params_from_stripe(%Stripe.Customer{} = customer) do
+  def to_params(%Stripe.Customer{} = customer) do
     customer
     |> Map.from_struct
     |> Map.take([:created, :currency, :delinquent, :id, :email])
     |> rename(:id, :id_from_stripe)
     |> keys_to_string
   end
```

Comment:
What about making `to_params` take `customer` and `attributes` so we don't have to do two separate adapter operations, then this function will combine the `add_non_stripe_attributes` in?